apps/src/templates/studioHomepages/teacherHomepageV2/sectionAvatars/section-avatars.module.scss

```diff
@@ -0,0 +1,149 @@
+@use '@code-dot-org/component-library-styles/typography.module.scss' as
+  typography;
+@import 'color';
+@import 'https://fonts.googleapis.com/css2?family=Noto+Color+Emoji&display=swap';
+
+// Section Avatars
+.sectionAvatarSmall {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 56px;
+  height: 56px;
+  border-radius: 50%;
+  font-family: 'Noto Color Emoji', sans-serif;
+  font-size: 32px;
+  margin-inline-end: 12px;
+}
+
+.sectionAvatarLarge {
+  display: flex;
```

Comment:
What do you think about de-duplicating everything that is not different between small large and xl into a general class.

i.e. move `display, justify-content` etc to a new class `.sectionAvatar` and then apply `.sectionAvatar` and `.sectionAvatarSize` to the component?